setting-up/workload-onboarding/container/index.md

```diff
@@ -1,63 +1,32 @@
 ---
-description: How to use the Bacalhau Docker image
+description: How to use the Bacalhau CLI in Docker
 ---
 
-# Bacalhau Docker Image
+# Bacalhau CLI from Docker
 
-This documentation explains how to use the Bacalhau Docker image to run tasks and manage them using the Bacalhau client.
+This documentation explains how to use Bacalhau CLI from docker.
 
 ## Prerequisites
 
-To get started, you need to install the Bacalhau client (see more information [here](../../../getting-started/installation.md)) and Docker.
+Install the [Bacalhau CLI in Docker](../../..//getting-started/installation#step-1.1-install-the-bacalhau-cli).
```

Comment:
Why do we need to install the bacalhau client at all? aren't we using the pre-built image?

Reply:
I had to install bacalhau client else I kept getting the error: ```rkodhand@blr-mpe5j ~ % docker run -t ghcr.io/bacalhau-project/bacalhau:latest \
    docker run \
        --id-only \
        --wait \
        ubuntu:latest \
        -- sh -c 'uname -a && echo "Hello from Docker Bacalhau!"'
12:02:36.828 | INF pkg/repo/fs.go:93 > Initializing repo at '/root/.bacalhau' for environment 'production'
publicapi: after posting request: "This endpoint is deprecated. See the migration guide at https://docs.bacalhau.org/references/cli-reference/command-migration for more information"```